index.html

```diff
@@ -204,55 +204,1951 @@ <h3>
+        </dd>
+        <dt>
+          <dfn>isLinear</dfn> member
+        </dt>
+        <dd>
+          `true` if the item represents a linear relationship between what is
+          measured and what is reported, or `false` if the data has been
+          processed in some way.
+        </dd>
+        <dt>
+          <dfn>isRange</dfn> member
+        </dt>
+        <dd>
+          `true` if the item assigns usages from a [=HID Usage=] range defined
+          by {{HIDReportItem/usageMinimum}} and {{HIDReportItem/usageMaximum}},
+          or `false` if the item has a sequence of [=HID Usage=] values in
+          {{HIDReportItem/usages}}.
+        </dd>
+        <dt>
+          <dfn>isVolatile</dfn> member
+        </dt>
+        <dd>
+          <p>
+            `true` if the item value can change without host interaction, or
+            `false` if the item value should only be changed by the host.
+          </p>
+          <p>
+            Only used for Feature and Output items.
+          </p>
+        </dd>
+        <dt>
+          <dfn>hasNull</dfn> member
+        </dt>
+        <dd>
+          `true` if the item has a null state in which it is not sending
+          meaningful data, or `false` if it does not have a null state. When in
+          a null state, the item will report a value outside of the specified
+          {{HIDReportItem/logicalMinimum}} and
+          {{HIDReportItem/logicalMaximum}}.
+        </dd>
+        <dt>
+          <dfn>hasPreferredState</dfn> member
+        </dt>
+        <dd>
+          `true` if the item has a preferred state to which it will return when
+          the user is not physically interacting with the control, or `false`
+          if the item does not have a preferred state.
+        </dd>
+        <dt>
+          <dfn>wrap</dfn> member
+        </dt>
+        <dd>
+          `true` if the item value rolls over when reaching either the extreme
+          high or low value, or `false` if the item value does not roll over.
+        </dd>
+        <dt>
+          <dfn>usages</dfn> member
+        </dt>
+        <dd>
+          <p>
+            If {{HIDReportItem/isRange}} is `true` or this item has no
+            associated [=HID Usage=] values, then {{HIDReportItem/usages}} MUST
+            be an empty sequence.
+          </p>
+          <p>
+            If {{HIDReportItem/isRange}} is `false` then
+            {{HIDReportItem/usages}} contains the sequence of [=HID Usage=]
+            values associated with this item.
+          </p>
+        </dd>
+        <dt>
+          <dfn>usageMinimum</dfn> member
+        </dt>
+        <dd>
+          <p>
+            If {{HIDReportItem/isRange}} is `true` then
+            {{HIDReportItem/usageMinimum}} contains the minimum [=HID Usage=]
+            value in the usage range associated with this item.
+          </p>
+          <p>
+            If {{HIDReportItem/isRange}} is `false` then
+            {{HIDReportItem/usageMinimum}} MUST be 0.
```

Comment:
Same here. We could use undefined instead of 0.

Reply:
Done. Before this lands we'll need to file spec issues against Chromium since these fields are currently treated as required.